pymc3/distributions/bound.py

```diff
@@ -14,40 +14,44 @@
 
 from numbers import Real
 
-import aesara.tensor as at
 import numpy as np
 
+from aesara.tensor.random.op import RandomVariable
+
 from pymc3.aesaraf import floatX
-from pymc3.distributions import transforms
+from pymc3.distributions import _logp
+from pymc3.distributions.continuous import BoundedContinuous
 from pymc3.distributions.dist_math import bound
-from pymc3.distributions.distribution import Continuous, Discrete, Distribution
+from pymc3.distributions.distribution import Continuous, Discrete
 
 __all__ = ["Bound"]
 
 
-class _Bounded(Distribution):
-    def __init__(self, distribution, lower, upper, default, *args, **kwargs):
-        self.lower = lower
-        self.upper = upper
-        self._wrapped = distribution.dist(*args, **kwargs)
+class BoundRV(RandomVariable):
+    name = "bound"
+    ndim_supp = 0
+    ndims_params = [0, 0, 0]
```

Comment:
Not sure about the `ndims_params`. In fact the distribution is characterized by the `original params + [0, 0]`. Also the `dtype` should be the same as the original... But I don't know what is the best approach here.

Reply:
I thought it wouldn't matter in this case since we are never actually drawing values of that shape in the first place. `original params + [0, 0]` will probably be a good idea to implement here if that's not the case.